backend/beyond_the_loop/models/prompts.py

```
@@ -0,0 +1,308 @@
+                    return False  # Bookmark was removed
+                else:
+                    db.execute(
+                        insert(user_prompt_bookmark).values(
+                            user_id=user_id,
+                            prompt_command=prompt_command
+                        )
+                    )
+                    db.commit()
+                    return True
+        
+        except Exception:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)